WebInstall/explainer_same_domain.md

```diff
@@ -61,7 +61,7 @@ const installApp = async () => {
 
 The **`navigator.install` method can overlap with some functionality of `beforeinstallprompt` for same-origin installation**. When the method is called it will trigger the UA to prompt for the installation of an application. This is analogous to when the end user clicks on an affordance that the UA might have to inform the user of installing. On Edge, Chrome (desktop) and Samsung Internet (mobile), this would be when the user clicks on the 'app available' banner or related UX that appears on the omnibox of the browser. For browsers that do not implement prompting, the expected behaviour is analogous to their installation paradigm. For example, in Safari (desktop) the behaviour might be that it shows a dialog to add the content to the dock as an app.
 
-On UAs that support prompting, the threshold for `navigator.install` to resolve on same-origin installations uses the same checks that `onbeforeinstallprompt` currently has for prompting (if required by the UA). The promise doesn't resolve unless the *installability criteria* is met. *Note that the criteria defined by UAs varies and can be that there is NO criteria*.
+On UAs that support prompting, the threshold for `navigator.install` to resolve on same-origin installations uses the same checks that `onbeforeinstallprompt` currently has for prompting (if required by the UA). The promise doesn't resolve unless the *installability criteria* is met. *Note that the criteria defined by UAs varies and can be that there is NO criteria aside from requiring an id in the site's manifest*.
 
 When called on the same domain, the **`install` method will trigger/open the prompt for installation the same way that using `onbeforeinstallprompt` does right now for browser that prompts.** If there is an error with the installation, then the promise returns a `DOMException` of type 'AbortError'. 
 
```

Comment:
Line 68 below:

> "Any same-origin content can be installed even if it is **NOT** an application."

This no longer appears to be true, if every same-origin signature expects an `id` to be explicitly defined in a manifest, then it is no-longer is possible to install non-applications.

I generally feel ok with that, as I would expect that it should be possible for the dev adding the usage of `navigator.install` to find a way to get a manifest added/updated, even if it's on a different section of the site.

If proceeding with this change, then this line needs to be modified or removed.

Reply:
Thanks! Removed